pkg/artifacts/signable.go

```diff
@@ -76,6 +86,18 @@ type image struct {
 	digest string
 }
 
+// StructuredSignable contains info for signable targets to become either subjects or materials in Intoto statements.
+// Name is the identifier used within TaskRun results to identify the target.
+// Uri is the resource uri for the target needed iff the target is a material.
+// Digest is the target's SHA digest.
+// Description describes some basic info of this signable target.
+type StructuredSignable struct {
+	Name        string
+	Description string
+	Uri         string
```

Comment:
```suggestion
	URI         string
```

https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
Done!